static/templates/recent_topic_row.hbs

```diff
@@ -0,0 +1,22 @@
+<tr id="{{stream_id}}:{{topic}}">
```

Comment:
Let's at least make it `recent_topic:{{stream_id}}:{{topic}}` to namespace these IDs.

Reply:
Done